docs/assets/js/charts.js

```diff
@@ -31,14 +34,11 @@ var extendedChartColors =
 	"#8bea12",
 	"#19dc80",
 	"#10d9be",
-	"#4c99e5",
-	"#c87adf",
-	"#ff5a99",
 ];
 
 function extendedChartColor(i)
 {
-	return extendedChartColors[(i + 3) % extendedChartColors.length];
+	return extendedChartColors[i % extendedChartColors.length];
```

Comment:
@pluehne I think this addition has no other purpose than rotating the color wheel, right?

Reply:
Exactly, and it shouldn’t be in the code.